manifests/v2/base/runtimes/pre-training/torch-distributed.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kubeflow.org/v2alpha1
+kind: ClusterTrainingRuntime
+metadata:
+  name: torch-distributed
+  labels:
+    training.kubeflow.org/phase: pre-training
+spec:
+  mlPolicy:
+    numNodes: 1
+    torch:
+      numProcPerNode: auto
+  template:
+    spec:
+      replicatedJobs:
+        - name: trainer-node
+          template:
+            spec:
+              template:
+                spec:
+                  containers:
+                    - name: trainer
+                      image: pytorch/pytorch:2.5.0-cuda12.4-cudnn9-runtime
+                      command:
+                        - /bin/bash
+                        - -c
+                        - |
+                          echo "Torch Distributed Runtime"
+
+                          echo "Torch Python Package"
+                          pip show torch
+
+                          echo "Torch Default Runtime Env"
+                          env | grep PET_
```

Comment:
Please let me know what do you think about it ?
Basically, I added it to allow users just see the default values of this runtime:
```
apiVersion: kubeflow.org/v2alpha1
kind: TrainJob
metadata:
  name: test-torch
spec:
  runtimeRef:
    apiGroup: kubeflow.org
    kind: ClusterTrainingRuntime
    name: torch-distributed-test
```

Reply:
Basically LGTM.

Btw, just a suggestion, we can also add `pip list` command to list packages in this container, making package version visible to users:)

Reply:
I think, `pip list` make sense, since it allows users to see all installed packages.